pkg/rpc/client/rpc.go

```diff
@@ -245,9 +245,31 @@ func (c *Client) GetNEP5Balances(address util.Uint160) (*result.NEP5Balances, er
 	return resp, nil
 }
 
-// GetNEP5Transfers is a wrapper for getnep5transfers RPC.
-func (c *Client) GetNEP5Transfers(address string) (*result.NEP5Transfers, error) {
+// GetNEP5Transfers is a wrapper for getnep5transfers RPC. Address parameter
+// is mandatory, while all the others are optional. Start and stop parameters
+// are supported since neo-go 0.77.0 and limit and page since neo-go 0.78.0.
+// These parameters are positional in the JSON-RPC call, you can't specify limit
+// and not specify start/stop for example.
+func (c *Client) GetNEP5Transfers(address string, start, stop *uint32, limit, page *int) (*result.NEP5Transfers, error) {
 	params := request.NewRawParams(address)
+	if start != nil {
+		params.Values = append(params.Values, *start)
+		if stop != nil {
+			params.Values = append(params.Values, *stop)
+			if limit != nil {
+				params.Values = append(params.Values, *limit)
+				if page != nil {
+					params.Values = append(params.Values, *page)
+				}
+			} else if page != nil {
+				return nil, errors.New("bad parameters")
+			}
+		} else if limit != nil || page != nil {
+			return nil, errors.New("bad parameters")
+		}
+	} else if stop != nil || limit != nil || page != nil {
```

Comment:
May be do this as first it as `start == nil && (.. || ..)` and shorten indent of if body?
The same could be done inside if body.

Reply:
That would duplicate `start` check which is what I've tried to avoid.